### PR TITLE
Adding DeepChemXAlphafold tutorial

### PR DESCRIPTION
**Pull request by soulpower007**
## Description
Adding DeepChemXAlphafold.ipynb notebook tutorial and added it in readme and respective csv as well. 
Fix #(issue)

## Type of change

Please check the option that is related to your PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
  - In this case, we recommend to discuss your modification on GitHub issues before creating the PR
- [ ] Documentations (modification for documents)

## Checklist

- [ ] My code follows [the style guidelines of this project](https://deepchem.readthedocs.io/en/latest/development_guide/coding.html)
  - [x ] Run `yapf -i <modified file>` and check no errors (**yapf version must be  0.32.0**)
  - [ x] Run `mypy -p deepchem` and check no errors
  - [ ] Run `flake8 <modified file> --count` and check no errors
  - [ x] Run `python -m doctest <modified file>` and check no errors
- [ x] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New unit tests pass locally with my changes
- [x ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: examples/tutorials/AlphafoldXDeepChem.ipynb [hunk 1]
```diff
@@ -0,0 +1,1626 @@
+{
```
Comment (rbharath) [inline examples/tutorials/AlphafoldXDeepChem.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #10.    
</pre><p>Make sure to remove extra whitespace</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (rbharath) [inline examples/tutorials/AlphafoldXDeepChem.ipynb:1]: <p>Combine these lines into a paragraph. Make sure to use complete sentences, active voice</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (rbharath) [inline examples/tutorials/AlphafoldXDeepChem.ipynb:1]: <p>Formatting errors. Also, again use complete sentences.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (rbharath) [inline examples/tutorials/AlphafoldXDeepChem.ipynb:1]: <p>Formatting</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (rbharath) [inline examples/tutorials/AlphafoldXDeepChem.ipynb:1]: <p>Formatting</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (rbharath) [inline examples/tutorials/AlphafoldXDeepChem.ipynb:1]: <p>Break this down into a few cells and add description between them</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (rbharath) [inline examples/tutorials/AlphafoldXDeepChem.ipynb:1]: <p>Too much code, let's break down into a few cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Comment (rbharath) [inline examples/tutorials/AlphafoldXDeepChem.ipynb:1]: <p>Formatting</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

File: examples/tutorials/DeepChemXAlphafold.ipynb [hunk 2]
```diff
@@ -0,0 +1,1767 @@
+{
```
Comment (rbharath) [inline examples/tutorials/DeepChemXAlphafold.ipynb:1]: <p>Why do we need this job name? Give a bit more context</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (rbharath) [inline examples/tutorials/DeepChemXAlphafold.ipynb:1]: <p>Move all install dependencies to the top</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (rbharath) [inline examples/tutorials/DeepChemXAlphafold.ipynb:1]: <p>Can you add a reference link to the A3M format specification?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (rbharath) [inline examples/tutorials/DeepChemXAlphafold.ipynb:1]: <p>Let's simplify and omit </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Comment (rbharath) [inline examples/tutorials/DeepChemXAlphafold.ipynb:1]: <p>Make this a "Save to Google Drive" section</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Comment (rbharath) [inline examples/tutorials/DeepChemXAlphafold.ipynb:1]: <p>Is there a tool we can use to visualize pockets?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>

Comment (soulpower007) [reply examples/tutorials/DeepChemXAlphafold.ipynb:1]: done

Comment (soulpower007) [reply examples/tutorials/DeepChemXAlphafold.ipynb:1]: done, removed the other option of adding custom mmseq files as it wasn't being used in this tutorial

Comment (soulpower007) [reply examples/tutorials/DeepChemXAlphafold.ipynb:1]: yes done

Comment (soulpower007) [reply examples/tutorials/DeepChemXAlphafold.ipynb:1]: sure restructured the tutorial

[the remark this answers is not in the document]
Comment (soulpower007) [reply examples/tutorials/DeepChemXAlphafold.ipynb:1]: yeup added explanations

[the remark this answers is not in the document]
Comment (soulpower007) [reply examples/tutorials/DeepChemXAlphafold.ipynb:1]: done

Comment (soulpower007) [reply examples/tutorials/DeepChemXAlphafold.ipynb:1]: nglview is commonly used to visualize proteins. However, it does't work properly on google colab. https://github.com/googlecolab/colabtools/issues/2853#issuecomment-1171699299. An issue was raised on it and google colab team has triaged it. We might have to look for alternatives of protein visualization on google colab.

[the remark this answers is not in the document]
Comment (soulpower007) [reply examples/tutorials/DeepChemXAlphafold.ipynb]: <p>sure done</p>

File: examples/tutorials/AlphafoldXDeepChem.ipynb (hunk 1 above)
Comment (soulpower007) [reply examples/tutorials/AlphafoldXDeepChem.ipynb]: <p>sure changed</p>

File: examples/tutorials/DeepChemXAlphafold.ipynb (hunk 2 above)
[the remark this answers is not in the document]
Comment (soulpower007) [reply examples/tutorials/DeepChemXAlphafold.ipynb]: <p>sure done</p>

**Review by rbharath — APPROVED** [review]
LGTM

**Pull request merged**